qcfractal/interface/collections/dataset.py

```diff
@@ -761,22 +912,7 @@ def list_contributed_values(self) -> List[str]:
 
         return list(self.data.contributed_values)
 
-    def get_contributed_values(self, key: str) -> ContributedValues:
-        """Returns a copy of the requested ContributedValues object.
-
-        Parameters
-        ----------
-        key : str
-            The ContributedValues object key.
-
-        Returns
-        -------
-        ContributedValues
-            The requested ContributedValues object.
-        """
-        return self.data.contributed_values[key.lower()].copy()
-
-    def get_contributed_values_column(self, key: str) -> 'Series':
```

Comment:
Deprecate get_contributed_values_column

Reply:
Low level and generally unused. Leaning towards just nuking it.

Reply:
Yeah, we can do that. Probably 0 uses in the wild.